packages/foam-vscode/src/core/model/note.ts

```diff
@@ -38,12 +38,17 @@ export interface Tag {
   range: Range;
 }
 
+export interface NoteBlock {
+  label: string;
+  range: Range;
+}
+
 export interface Resource {
   uri: URI;
   type: string;
   title: string;
   properties: any;
-  // sections: NoteSection[]
+  blocks: NoteBlock[];
```

Comment:
Calling this `blocks` so that in the future it will also house inline blocks (https://github.com/foambubble/foam/issues/294) - sections are just a type of block

Reply:
I see your point about blocks, it is inspired by names we have used before, but I also don't love it.
I am not sure about Section as I don't think a section represents well an "inline block".. mm, I am torn about it.
It's almost an Anchor, but that is a point, not a range.
I guess it could also be considered a Fragment, but that feels confusing with `URI.fragment`.

Other ideas?

Reply:
decided to go for "section"